CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ New modules
 
 * The following new modules have been added to the library:
   ```
+  Algebra.Properties/Semigroup
+
```

Comment:
Please use only `.` as a separator in the name of the new modules

Reply:
Yes, a typo.